app/router.js

```diff
@@ -120,6 +120,9 @@ router.map(function() {
     this.route('users', function() {
       this.route('list', { path: '/:users_status' });
       this.route('user', { path: '/u/:user_id' });
+      this.route('user-sessions', function() {
```

Comment:
What is this `/user-sessions` route for ? 

Sessions for a user should be located at `/users/<id>/sessions/<session_status>`

Reply:
@niranjan94 now I have created the route as /users/sessions/<user_id>/<session_status> as for creating it as /users/<id>/sessions/<session_status> I had to introduce some other path in between.